man/man1/p11sak.1.in

```diff
@@ -432,3 +437,40 @@ prints help for the usage of
 .B p11sak
 and/or the respective command.
 .PP
+.
+.
+.
+.SH "p11sak_defined_attrs.conf"
+In the new output config file a user can define additional attributes, which are not 
```

Comment:
Why the "new"?  The man-page should not include such form of history.

Reply:
Done